backend/LexData/Migrations/20240606084230_AddOrgProjectsTable.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LexData.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddOrgProjectsTable : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "OrgProjects",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(type: "uuid", nullable: false),
+                    OrgId = table.Column<Guid>(type: "uuid", nullable: false),
+                    ProjectId = table.Column<Guid>(type: "uuid", nullable: false),
+                    CreatedDate = table.Column<DateTimeOffset>(type: "timestamp with time zone", nullable: false, defaultValueSql: "now()"),
+                    UpdatedDate = table.Column<DateTimeOffset>(type: "timestamp with time zone", nullable: false, defaultValueSql: "now()")
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_OrgProjects", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_OrgProjects_Orgs_OrgId",
+                        column: x => x.OrgId,
+                        principalTable: "Orgs",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                    table.ForeignKey(
+                        name: "FK_OrgProjects_Projects_ProjectId",
+                        column: x => x.ProjectId,
+                        principalTable: "Projects",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateIndex(
+                name: "IX_OrgProjects_OrgId_ProjectId",
+                table: "OrgProjects",
+                columns: new[] { "OrgId", "ProjectId" },
+                unique: true);
+
+            migrationBuilder.CreateIndex(
+                name: "IX_OrgProjects_ProjectId",
+                table: "OrgProjects",
+                column: "ProjectId");
+        }
```

Comment:
I have no idea why the migration created a ProjectId index but not an OrgId index. As far as I can tell, I never asked for a ProjectId index. Anyone know why this index was autocreated but no corresponding OrgId index was created?

Reply:
it may have created the index due to the query filter but I doubt it. I can't tell, though I would guess something is missing somewhere, but I can't tell.

Reply:
Commit 2ad2ebbd should fix this. Note that I edited the migration by hand, so anyone who already applied that migration to their code (such as by checking out this branch and running it) will get EF Core confused: it will think the migration has already been applied, but in fact the OrgId index won't exist. That's not likely to be an issue for anyone but me, and I can easily fix that by deleting my DB volume so that it gets recreated from scratch.

**WARNING:** I've been unable to test it because https://github.com/sillsdev/languageforge-lexbox/issues/871 is preventing me from building this branch at the moment. So if I've made mistakes on the migration edits in 2ad2ebbd, I'm unable to catch them right now. Please double-check 2ad2ebbd carefully.

Leaving conversation unresolved until someone else can confirm that 2ad2ebbd is safe.

Reply:
Okay, I've finally gotten the issues with local-dev ironed out and I can confirm that commit https://github.com/sillsdev/languageforge-lexbox/commit/2ad2ebbd645fff7485cfcbcba1844f2e80f1c981 is just fine. So I'm resolving this conversation.